frontend/src/App.test.tsx

```diff
@@ -1,34 +1,46 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
 import App from './App';
-import { Button } from './components/button/Button';
+import Counter from './components/counter/Counter';
 import userEvent from '@testing-library/user-event';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import counterReducer from './reducers/counterReducer';
 
 describe('App', () => {
   it('renders App component', () => {
     render(<App />);
-    screen.debug();
     expect(screen.getByText(/Welcome to project/i)).toBeInTheDocument();
   });
 });
 
-describe('Button', () => {
-  it('renders Button', () => {
-    render(<Button />);
-    screen.debug();
-    expect(screen.getByText(/Большая зеленая кнопка/i)).toBeInTheDocument();
-    expect(screen.getByRole('button')).toBeInTheDocument();
-  });
-});
+const renderWithRedux = (
+  component: any,
+  { initialState, store = createStore(counterReducer, initialState) } = {}
```

Comment:
Regarding errors you showed me yesterday: in general it was caused by the fact that `initialState` here is implicitly set as any.
In general, to fix that issue you can do next:
```
import { CounterState } from './actions/index';
import {createStore, Store} from 'redux'

type ReduxWrapper = {
  initialState?: CounterState,
  store?: Store<CounterState>
}

const renderWithRedux = (
  component: any,
  { initialState, store = createStore(counterReducer, initialState) }: ReduxWrapper = {}
) => {
```
It's probably not ideal solution, but it's used for testing purposes, so we're ok here, and it a test for code that will be removed in future.
So, you don't have to apply suggested changes.